libs/translations-finder/src/lib/translations-finder.util.ts

```diff
@@ -0,0 +1,243 @@
+import * as glob from 'glob';
+import * as fs from 'fs';
+
+/**
+ * @description
+ * Extracts the string key of the translation from the text
+ *
+ * @method extractString
+ *
+ * @param {string} translation
+ * The whole text matched by regex
+ *
+ * @param {RegExp} translationKeyRegex
+ * The regex to extract the translation key from the matched text
+ *
+ * @return {string}
+ * Extracted string
+ */
+const extractString = (translation: string, translationKeyRegex: RegExp): string => {
+  return (translation.match(translationKeyRegex) || [])[0];
```

Comment:
what's this OR for ?

Reply:
![Screenshot from 2019-12-16 16-17-45](https://user-images.githubusercontent.com/49659466/70918627-aacf2a80-201f-11ea-89ad-29246845309b.png)
I needed it not to have this error in case user chose regex for the text that doesn't match anything. I could use try-catch block instead.

Reply:
![Screenshot from 2019-12-16 16-29-01](https://user-images.githubusercontent.com/49659466/70919458-2f6e7880-2021-11ea-85ac-cec6a757c285.png)
I needed it not to have this error in case user chose regex for the text that doesn't match anything. I could use try-catch block instead.